.github/actions/devcontainer_run_command/action.yml

```diff
@@ -178,6 +178,15 @@ runs:
     - name: Run command in DevContainer
       shell: bash
       run: |
+        # Write command to a temporary script file with lowercase name
```

Comment:
Do we care about the file name being in lower case?

Reply:
yes, it doesn't work otherwise on mounting